input/haleu/sensitivity-analysis/oat/mmr/mmr_share_driver.py

```diff
@@ -40,8 +40,7 @@
                                           duration,
                                           reactor_prototypes,
                                           demand_equation,
-                                          'MMR',
-                                          int(params['mmr']))
+                                          {'MMR':int(params['mmr'])})
```

Comment:
The dictionary here defines the build share percent (the value) for each specified prototype (the key). This change is because of the updates to the `determine_deployment_scheme` function in `scripts/create_AR_DeployInst.py` in PR #143. This does not change the lifetime or any parameters about the MMR prototype.